packages/e2e_tests/cypress/support/customActions/footer-link.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+	clickFooterLink: (linkType) => {
+		switch (linkType) {
+			case 'Sitemap':
+				cy.get('[data-cy="Privacy"]').click();
+				break;
+			case 'Terms and conditions':
+				cy.get('[data-cy="Terms and conditions"]').click();
+				break;
+			case 'Accessibility':
```

Comment:
Would be so easy to make these variables and make it generic

Reply:
I will look into it